docs/jwst/outlier_detection/arguments.rst

```diff
@@ -50,59 +75,53 @@ that control the behavior of the processing:
 ``--backg``
   User-specified background value to apply to the median image.
 
-``--rolling_window_width``
-  Number of integrations over which to take the median when using rolling-window
-  median for TSO observations.
-
-``--save_intermediate_results``
-  Specifies whether or not to save any intermediate products created
-  during step processing.
-
 ``--resample_data``
   Specifies whether or not to resample the input images when
   performing outlier detection.
 
-``--good_bits``
-  The DQ bit values from the input image DQ arrays
-  that should be considered 'good' when building the weight mask. See
-  DQ flag :ref:`dq_parameter_specification` for details.
-
 ``--allowed_memory``
   Specifies the fractional amount of
   free memory to allow when creating the resampled image. If ``None``, the
   environment variable ``DMODEL_ALLOWED_MEMORY`` is used. If not defined, no
   check is made. If the resampled image would be larger than specified, an
   ``OutputTooLargeError`` exception will be generated.
-
   For example, if set to ``0.5``, only resampled images that use less than half
   the available memory can be created.
 
 ``--in_memory``
   Specifies whether or not to load and create all images that are used during
   processing into memory. If ``False``, input files are loaded from disk when
   needed and all intermediate files are stored on disk, rather than in memory.
-  This flag is superseded by the pipeline-level ``--in-memory`` flag, and thus
-  has no effect when running the full level 3 pipeline.
+  Has no effect for spectroscopic data, and is superseded by the pipeline-level
+  ``in_memory`` parameter set by ``calwebb_image3``.
+
```

Comment:
You might want to split this sentence into two.  Is this what you mean: 
Has no effect for spectroscopic data. For imager data it is superseded by the pipeline-level  ``in_memory`` parameter set by ``calwebb_image3``.

Reply:
thanks, yes that is what I mean.